### PR TITLE
task/WG-191: do not use headers for analytics information due to WG-191

### PR DESCRIPTION
**Pull request by nathanfranklin**
## Overview: ##

Disable added headers and use query params on single route instead. Headers can't be used due to https://tacc-main.atlassian.net/browse/WG-191 . The use of headers is left in code but commented out; to be re-used in WG-192 when we start using Taips V3.

See https://github.com/TACC-Cloud/geoapi/pull/161 for more details.

## Related Jira tickets: ##

* [WG-140](https://tacc-main.atlassian.net/browse/WG-140)
* [WG-191](https://tacc-main.atlassian.net/browse/WG-191)

## Testing Steps: ##
See https://github.com/TACC-Cloud/geoapi/pull/161

### CODE REVIEW

**Review by sophia-massie — APPROVED** [review]
LGTM! 🦖

File: angular/src/app/app.interceptors.ts [hunk 1]
```diff
@@ -38,29 +38,34 @@ export class JwtInterceptor implements HttpInterceptor {
...
-      });
+
+      // Using query params instead of custom headers due to https://tacc-main.atlassian.net/browse/WG-191
+      let analytics_params = {};
+
+      analytics_params = { ...analytics_params, application: 'hazmapper' };
+
+      // for guest users, add a unique id
+      if (!this.authSvc.isLoggedIn()) {
+        // Get (or create of needed) the guestUserID in local storage
```
Comment (taoteg) [inline angular/src/app/app.interceptors.ts:51]: typo: "or create _if_ needed..."

**Review by taoteg — APPROVED** [review]
LGTM! Did not run locally, just reviewed in PR. One minor typo identified in the comments.

File: angular/src/app/app.interceptors.ts (hunk 1 above)
Comment (nathanfranklin) [reply angular/src/app/app.interceptors.ts]: thanks. fixed.

**Pull request merged**